### DISCUSSION TITLE
Getting signed in by just the share URL (Critical Bug?)

### DISCUSSION THREAD

**Discussion by ShubhamVerma1811** [umami-software/umami#1741] [Q&A]
Steps to reproduce
1. Open any share URL in incognito
2. Click on Realtime from header
3. Now go to home

The user is somehow logged in to the account by doing this.

**Comment by briancao** [comment]
We have this reported as an issue. https://github.com/umami-software/umami/issues/1720

Currently working on it.  There will likely be a security release later this week as a final release for v1.
